e2e/specs/st_map.spec.js

```diff
@@ -19,43 +19,44 @@ describe("st.map", () => {
     cy.loadApp("http://localhost:3000/");
 
     // Wait for all the maps to be loaded.
-    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 7);
+    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 4);
   });
 
-  it("displays 7 maps", () => {
-    cy.get(".element-container .stDeckGlJsonChart").should("have.length", 7)
+  it("displays 4 maps", () => {
```

Comment:
Could you please leave a small comment in this thread why the number here changes as it is not obvious why adding support for Modin / Snowpandas changes tests in `st.map`. We discussed this offline and you also mention it on a high-level in the PR description, but I think it would be valuable to explain it on a concrete example like this for future reference

Reply:
I removed all e2e tests related to our mock classes since it doesn't really add any value to also do that via e2e tests.